### PR TITLE
Wave array in Python

### PR DESCRIPTION
**Pull request by Aayushi-Mittal**
### Have you read the [Contributing Guidelines on Pull Requests](https://github.com/TesseractCoding/NeoAlgo/blob/master/CONTRIBUTING.md#pull-requests)?

Yes

### Description

Sort the array into a wave-like array and return it. In other words, arrange the elements into a sequence such that a1 >= a2 <= a3 >= a4 <= a5..... (considering the increasing lexicographical order) in python.
For eg: arr[] = {1,2,3,4,5}
Output: 2 1 4 3 5

### Checklist

- [x] I've read the contribution guidelines.
- [x] I've checked the issue list before deciding what to submit.
- [x] I've edited the `README.md` and link to my code.

## Related Issues or Pull Requests

#6315

### CODE REVIEW

**Comment by Aayushi-Mittal** [comment]
@ankitaggarwal23 @HarshCasper Please review my pull request.

**Comment by Aayushi-Mittal** [comment]
@ankitaggarwal23 Please merge the PR if everything is fine.

**Comment by Aayushi-Mittal** [comment]
Otherwise, It will become unassigned. If possible please add pinned label to the issue.

**Comment by Aayushi-Mittal** [comment]
@ankitaggarwal23 Please merge the PR if everything is fine.

File: Python/README.md [hunk 1]
```diff
@@ -405,6 +405,7 @@ _add list here_
 - [Union of two arrays](other/Union_of_two_arrays.py)
 - [Array Rolation](other/Program_to_rotation_array.py)
 - [Tower of Hanoi](other/Tower_of_Hanoi.py)
+- [Wave Array](/other/Wave_Array.py)
```
Comment (ankitaggarwal23) [inline Python/README.md]: Remove / before other

File: Python/other/Wave_Array.py [hunk 2]
```diff
@@ -0,0 +1,34 @@
+"""
+Sorting the array into a wave-like array. In other words, arranging the elements into a sequence such that a1 >= a2 <= a3 >= a4 <= a5..... (considering the increasing lexicographical order) in python.
+For eg: arr[] = {1,2,3,4,5}
+Output: 2 1 4 3 5
+"""
```
Comment (ankitaggarwal23) [inline Python/other/Wave_Array.py]: Make it short and understandable

File: Python/other/Wave_Array.py [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+    return A
+
+A = list(map(int,input("Enter the elements of the array : ").split()))
+N = len(A)
+
+A = convertToWave(N,A)
+
+print("Wave Array: ")
+for el in A:
+  print(el, end=" ")
```
Comment (ankitaggarwal23) [inline Python/other/Wave_Array.py]: Don't use a single letter for variable names.

**Review by ankitaggarwal23 — CHANGES_REQUESTED** [review]
Few changes: 

- Add Multi-Line Comments 
- Add a beginner-friendly description at the top
- Add a new line at the EOF
- Use proper display message how you are reading and writing outputs

**Comment by Aayushi-Mittal** [comment]
Did the required changes. Please review.

**Review by ankitaggarwal23 — APPROVED** [review]

**Pull request merged**